frontend/src/components/Search.module.scss

```diff
@@ -4,6 +4,10 @@
   height: 37px !important;
 }
 
-.wide {
+.smallwide {
```

Comment:
:nitpick: there are many sizes available in the USWDS system

```
.font-sans-3xs
.font-sans-2xs
.font-sans-xs
.font-sans-sm
.font-sans-md
.font-sans-lg
.font-sans-xl
.font-sans-2xl
.font-sans-3xl

```

Reply:
Here we are trying to control the width of the search bar and not the size of the text inside of it.